ods_ci/tests/Tests/100__deploy/100__installation/110__model_serving_watsonx.robot

```diff
@@ -0,0 +1,101 @@
+*** Settings ***
```

Comment:
As we said this can possibly be moved to the operator reach test cases, but we'll talk more about it offline. For now it should not cause issues having this code running as well